PubHub/PubHub.API/Controllers/BooksController.cs

```diff
@@ -446,8 +464,14 @@ public async Task<IResult> UpdateBookAsync(Guid id, [FromBody] BookUpdateModel u
         [HttpDelete("{id}")]
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound, Type = typeof(ProblemDetails))]
-        public async Task<IResult> DeleteBookAsync(Guid id)
+        public async Task<IResult> DeleteBookAsync(Guid id, [FromHeader] string appId)
         {
+            if (!_whitelistService.TryVerifyApplicationAccess(appId, GetType().Name, out IResult? problem))
+                return problem;
+
+            //if (_whitelistService.Try(out IResult problem))
+            //  return problem;
```

Comment:
Do you want an outbreak? This is how The Walking Dead: Code Edition begins.

Reply:
Whoops forgot to remove my "playing around with ideas" code 😅